extensions/wikia/Helios/User.class.php

```diff
@@ -2,24 +2,31 @@
 
 namespace Wikia\Helios;
 
+use Wikia\Logger\WikiaLogger;
+
 /**
  * A helper class for dealing with user-related objects.
  */
 class User {
 
-	const ACCESS_TOKEN_COOKIE = 'access_token';
+	const ACCESS_TOKEN_COOKIE_NAME = 'access_token';
+
+	// This is set to 6 months, 6*30*24*60*60 = 15552000
+	const ACCESS_TOKEN_COOKIE_TTL = 15552000;
```

Comment:
In Mercury, the cookie TTL is 15778500 seconds, which is 182.621527777777 days. I guess someone wanted 6 months to be (very) precisely half a year. Anyway, I think the expiration for the cookie for the two platforms should be equal, so we should either make it 15778500 here or change it in Mercury.